### PR TITLE
[ALCS-144] 4 Date date picker for meetings and site visits

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: alcs-api/src/application/application-meeting/application-meeting.service.ts [hunk 1]
```diff
@@ -51,17 +52,63 @@ export class ApplicationMeetingService {
...
+    if (meeting.meetingEndDate !== undefined) {
+      existingMeeting.meetingPause.endDate = meeting.meetingEndDate
+        ? new Date(meeting.meetingEndDate)
+        : null;
+    }
+
+    if (
+      meeting.reportStartDate !== undefined ||
+      meeting.reportEndDate !== undefined
+    ) {
```
Comment (mhuseinov) [inline alcs-api/src/application/application-meeting/application-meeting.service.ts:83]: maybe ?
if ( meeting.reportStartDate || meeting.reportEndDate)

Comment (dhaselhan) [reply alcs-api/src/application/application-meeting/application-meeting.service.ts:83]: If they are null I need to update the value to null, if they are undefined I dont want to change it

File: alcs-frontend/src/app/shared/utils/date-format.ts [hunk 2]
```diff
@@ -11,5 +11,6 @@ export const DATE_FORMATS = {
     monthYearA11yLabel: 'YYYY',
     monthLabel: 'MMM',
     monthDayLabel: 'MMM-DD',
+    popupHeaderDateLabel: 'YYYY-MMM-DD',
```
Comment (mhuseinov) [inline alcs-frontend/src/app/shared/utils/date-format.ts:14]: 👍

**Review by mhuseinov — APPROVED** [review]
👍

**Review by mhuseinov — APPROVED** [review]

File: alcs-api/src/utils/incoming-date.formatter.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const formatIncomingDate = (date?: number) => {
```
Comment (mhuseinov) [inline alcs-api/src/utils/incoming-date.formatter.ts:1]: numberToDate() ?

**Pull request merged**